schemas/api_schema.js

```diff
@@ -127,10 +127,15 @@ type Mutation {
                 check_sessions_count(count);
                 return db.Session.findAll({ offset: start, limit: count });
             },
-            SessionsByUser: (parent, { id, start, count }, { db }, info) => {
+            SessionsByUser: (
+                parent,
+                { user_id, start, count },
```

Comment:
FYI if anyone looks at this PR, here is the bug - parameter changed from id to user_id